python_fundamentals-master/07_classes_objects_methods/07_01_car.py

```diff
@@ -10,20 +10,24 @@
 
 '''
 
-class car():
+class Car():
     def __init__(self, model, year, max_speed):
         self.model = model
         self.year = year
         self.max_speed = int(max_speed)
```

Comment:
I'm not sure why you need to cast this to `int()` here -- if anything you might want to put in the document string "max_speed: this is an `int` that should be blah blah" - basically people have to learn how to use your class correctly and you may want to provide them with some documentation for that.
But casting it to int? This would only really work if they entered a string.
why would they even do that?